python/lsst/drp/tasks/reprocess_visit_image.py

```diff
@@ -0,0 +1,581 @@
+        doc="Task to generate sky sources ('empty' regions where there are no detections).",
+    )
+    deblend = pexConfig.ConfigurableField(
+        target=lsst.meas.deblender.SourceDeblendTask, doc="Split blended sources into their components."
+    )
+    measurement = pexConfig.ConfigurableField(
+        target=lsst.meas.base.SingleFrameMeasurementTask,
+        doc="Task to measure sources to return in the output catalog.",
+    )
+    normalized_calibration_flux = pexConfig.ConfigurableField(
+        target=lsst.meas.algorithms.NormalizedCalibrationFluxTask,
+        doc="Task to normalize the calibration flux (e.g. compensated tophats).",
+    )
+    apply_aperture_correction = pexConfig.ConfigurableField(
+        target=lsst.meas.base.ApplyApCorrTask,
+        doc="Task to apply aperture corrections to the measured sources.",
+    )
+    set_primary_flags = pexConfig.ConfigurableField(
+        target=lsst.meas.algorithms.setPrimaryFlags.SetPrimaryFlagsTask,
+        doc="Task to add isPrimary to the catalog.",
+    )
+    catalog_calculation = pexConfig.ConfigurableField(
+        target=lsst.meas.base.CatalogCalculationTask,
+        doc="Task to compute catalog values using only the catalog entries.",
+    )
+    post_calculations = pexConfig.ConfigurableField(
+        target=lsst.meas.base.SingleFrameMeasurementTask,
+        doc="Task to compute catalog values after all other calculations have been done.",
+    )
+    calib_match_radius = pexConfig.Field(
+        dtype=float,
+        default=0.2,
+        doc="Radius in arcseconds to cross-match calib_sources to the output catalog.",
+    )
+
+    def setDefaults(self):
+        super().setDefaults()
+
+        # No need to redo background: we have the global background model.
+        self.detection.reEstimateBackground = False
+        self.detection.doTempLocalBackground = False
+
+        # TODO: Note: these apertures were selected for HSC, and may not be
+        # what we want for LSSTCam.
+        self.measurement.plugins["base_CircularApertureFlux"].radii = [
+            3.0,
+            4.5,
+            6.0,
+            9.0,
+            12.0,
+            17.0,
+            25.0,
+            35.0,
+            50.0,
+            70.0,
+        ]
+        lsst.meas.extensions.shapeHSM.configure_hsm(self.measurement)
+        self.measurement.plugins.names |= ["base_Jacobian", "base_FPPosition", "ext_photometryKron_KronFlux"]
+        self.measurement.plugins["base_Jacobian"].pixelScale = 0.2
+
+        # TODO: should make this the ApertureFlux default!
+        # Use a large aperture to be independent of seeing in calibration
+        self.measurement.plugins["base_CircularApertureFlux"].maxSincRadius = 12.0
+
+        # Only apply calibration fluxes, do not measure them.
+        self.normalized_calibration_flux.do_measure_ap_corr = False
+
+        # TODO: hopefully we can get rid of this, once Jim sorts out SDM.
```

Comment:
I am still not sure what this is referring to.

Reply:
I'll remove it, since that cleanup is probably not going anywhere for a long time.